### PR TITLE
hep: handle files properly

### PR DESCRIPTION
**Pull request by michamos**
I didn't add the keys I asked about in #200 (`figures/name` and `figures/type`), if they are needed I will do an update.

### CODE REVIEW

**Comment by david-caro** [comment]
Yep, they are needed.

**Comment by michamos** [comment]
@david-caro I updated the PR taking into account our discussion IRL. Could you have a look?

File: inspire_schemas/records/hep.yml [hunk 1]
```diff
@@ -650,6 +629,56 @@ properties:
...
+                    description: |-
+                        :MARC: ``8564_y``
+
+                        Describes how this document relates to the document that
+                        this record represents.
+                        :example: ``table of contents``
+                    type: string
+                fulltext:
+                    title: Whether this document contains the fulltext
+                    type: boolean
```
Comment (david-caro) [inline inspire_schemas/records/hep.yml]: Do we want to put anything that's not the fulltext here?

**Comment by michamos** [comment]
yes, parts of the fulltext, e.g. only a table of contents, or related material that is not the fulltext (poster presented at a conference attached to the conference paper).

**Pull request merged**